packages/manager/apps/dedicated/client/app/network-security/scrubbing-center/scrubbing-center.controller.js

```diff
@@ -98,20 +111,50 @@ export default class ScrubbingCenterController extends AbstractCursorDatagridCon
         )
         .then((data) => {
           this.ipsList = data.map(({ ipBlock }) => ipBlock);
+          this.isEmpty = !this.ipsList.length;
           this.selectedIp = this.ipsList;
           this.getAllEvents();
         });
     } else {
       this.isServiceSelected = false;
+      this.getAllEvents();
     }
   }
 
   checkSelectedIp(value) {
+    this.isValid = true;
     if (!value) {
       return null;
     }
 
+    if (value.indexOf('/') === -1 && !ipaddr.isValid(value)) {
+      this.isValid = false;
+      return null;
+    }
+
+    if (value.indexOf('/') > -1) {
+      const ip = value.split('/');
+      if (!ipaddr.isValid(ip[0])) {
+        this.isValid = false;
+      }
+      console.log('nan', isNaN(ip[1]));
+      if (!isNaN(ip[1])) {
+        this.isValid = false;
+      }
+    }
+    if (!this.isValid) {
+      return null;
+    }
+
     this.selectedIp = value;
+    if (value.indexOf('.') > -1 && value.indexOf('/') === -1) {
+      this.selectedIp = `${value}/32`;
+      this.model = this.selectedIp;
+    }
+    if (value.indexOf(':') > -1 && value.indexOf('/') === -1) {
+      this.selectedIp = `${value}/128`;
+      this.model = this.selectedIp;
+    }
     return this.getAllEvents();
   }
```

Comment:
no totally agree with your comment. because if the value is empty, we don't need to display the error message, we need this message to explain that the IP set is wrong.